### DISCUSSION TITLE
Support for Wireguard networking instead of Zerotier

### DISCUSSION THREAD

**Discussion by FLUPS** [tobychui/zoraxy#409] [Ideas]
Zerotier as VPN and network service is fine but not free of charge, how about an easy implementation for Wireguard connections ?

**Comment by tobychui** [comment]
In fact I am considering removal of ZeroTier (and any kind of GAN) all together. Though, before that, I will need a proper working plugin system in place. If you got the time, feel free to contribute some code :)
